src/api/lib/tasks/make_user_an_admin.rake

```diff
@@ -17,4 +17,23 @@ namespace :user do
 
     exit
   end
+
+  task remove_admin_rights: :environment do
```

Comment:
Hm, can't you make a toggle out of this?

Reply:
I've been thinking about this and I like it better than the existing solution. Check it out again, please.